Makefile

```diff
@@ -0,0 +1,26 @@
+container_exists := `docker ps -a -q -f name=SPA`
+dir := `pwd`
+
+up:
+	if [ ${container_exists} ] ; then \
+		docker stop SPA ; \
+		docker rm SPA ; \
+	fi
+	docker run -d --name SPA --mount type=bind,source="${dir}/codebase",target=/codebase -p 5000:5000 spa
+
+exec:
+	docker exec -tiu root SPA /bin/sh
+
+stop:
+	docker stop SPA
+
+build:
+	if [ ${container_exists} ] ; then \
```

Comment:
Build command should not stop container

Reply:
Pls explain why you think so. This is one entry for both build and rebuild. And for rebuild I need to recreate containers. First stop then up.

Reply:
I did not make separate command for a simple build this is build which ends up with new containers up.